tools/pipelines/templates/include-build-tools.yml

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# include-install-build-tools
+#
+# This template can be included in pipelines to install the Fluid build-tools locally so flub can be used in the
+# pipeline.
+
+parameters:
+- name: buildDirectory
+  type: string
+
+- name: buildToolsVersionToInstall
+  type: string
+  default: repo
+
+# The path to the pnpm store.
+- name: pnpmStorePath
+  type: string
+  default: $(Pipeline.Workspace)/.pnpm-store
+
+steps:
```

Comment:
These steps are outdated. If you copied my files from my PR, sorry, that PR is old and wasn't updated with the correct patterns. Anyway, you want this to look like [the	logic in include-set-package-versions](https://github.com/microsoft/FluidFramework/blob/main/tools/pipelines/templates/include-set-package-version.yml#L48). Ideally that script would also be updated to use this new template, so there is still only one template to maintain.